cpp/ql/test/library-tests/dataflow/dataflow-tests/test.ql

```diff
@@ -1,5 +1,27 @@
 import DataflowTestCommon
 
-from DataFlow::Node sink, DataFlow::Node source, TestAllocationConfig cfg
-where cfg.hasFlow(source, sink)
-select sink, source
+class ASTDataFlowTest extends InlineExpectationsTest {
```

Comment:
This class (and the IR version) looks a lot like the `ASTFieldFlowTest` class we have in the `fields` directory. At some point, it'd be nice to unify them (but that doesn't have to be in this PR)